1-reql-language/geo-support/python.md

```diff
@@ -107,7 +107,7 @@ While there *is* a [circle] command, it approximates a circle by defining either
 
 # Geospatial indexes #
 
-To create indexes on fields containing geometry objects, you simply use the standard [index_create](/api/python/index_create/) command, setting the `geo` optional argument to `Prue`. In JavaScript, this would be:
+To create indexes on fields containing geometry objects, you simply use the standard [index_create](/api/python/index_create/) command, setting the `geo` optional argument to `True`. In Python, this would be:
 
 ```py
 r.table('sites').index_create('locations', {geo: true})
```

Comment:
On an unrelated note, It looks like the Python and Ruby examples are not properly translated (the opt arg style is wrong). Pinging @chipotle